tests/testthat/test-pmcmc.R

```diff
@@ -75,7 +75,8 @@ test_that("pmcmc runs without error", {
                     dimnames = list(
                       c('beta', 'start_date'),
                       c('beta', 'start_date')
-                    ))
+                    )), 
+   output_proposals = TRUE
```

Comment:
Add a quick test on the shape of output_proposals?

Reply:
sorry, I misunderstood this! adding a check now!

Reply:
all sorted.